var/spack/repos/builtin/packages/py-lxml/package.py

```diff
@@ -30,3 +31,6 @@ class PyLxml(PythonPackage):
     depends_on('py-html5lib', when='+html5', type=('build', 'run'))
```

Comment:
The libxml2 and libxslt dependencies should also be `type='link'` if we want to use Spack's compiler wrappers to locate and RPATH them. In that case, `AUTO_RPATH` may not even be needed. Can you try removing `AUTO_RPATH` and add `link` and see if this works?

Reply:
Seems like link works as well. I made the change.